estate/models/estate_property.py

```diff
@@ -0,0 +1,32 @@
+from odoo import fields, models
+
+
+class Property(models.Model):
+    _name = "estate.property"
+    _description = "Real Estate Application"
+
+    name=fields.Char(string="Name",required=True)
```

Comment:
```suggestion
    name = fields.Char(string="Name", required=True)
```
Don't hesitate to give space to your code, meaning an '=' when declaring a field or a variable is always separated with two empty spaces, and a coma is followed by a space as well. This comment applies to all field below as well